src/Common/src/Interop/OSX/Interop.CoreFoundation.CFData.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+// Declared as signed long, which has sizeof(void*) on OSX.
+using CFIndex=System.IntPtr;
+
+internal static partial class Interop
+{
+    internal static partial class CoreFoundation
+    {
+        [DllImport(Libraries.CoreFoundationLibrary)]
+        private static extern unsafe byte* CFDataGetBytePtr(SafeCFDataHandle cfData);
+
+        [DllImport(Libraries.CoreFoundationLibrary)]
+        private static extern CFIndex CFDataGetLength(SafeCFDataHandle cfData);
+
+        internal static byte[] CFGetData(SafeCFDataHandle cfData)
```

Comment:
I changed CFStringToString to use GetBytePtr and GetLength; since they're in the same interop class it already had access.